admin/create-dumps.sh

```diff
@@ -32,6 +32,13 @@ function add_rsync_include_rule {
     echo "$SHA256_FILE_RULE" >> "$RULE_FILE"
 }
 
+if [[ "${CONTAINER_NAME}" = "listenbrainz-cron-prod" && "${PROD}" = "beta" ]]
```

Comment:
what does the `"${PROD}" = "beta"` check do here?


when we talked about this, I was considering installing separate configs in consul - e.g. instead of `docker-server-configs/LB/create-dumps-config.json` have `docker-server-configs/LB/create-dumps-config.prod.json` and `.beta.json`. This would let us ensure that we never load prod settings in test or beta, even if we accidentally start the cron container in those environments.


One issue with this check is that it makes it difficult to run this script either locally for testing, or in the one-off case where we might want to run it in test. It involves specifically setting some environment variables only in this situation. Whereas having separate constants in consul/config.sh would allow us to just enable a flag to allow us to run the dump

Reply:
Whops, it should be "prod" -- leftover from testing.

I agree that we need to have better testing for this -- let's add a ticket for that, but continue on with this for the time being.

Reply:
https://tickets.metabrainz.org/browse/LB-819